project/SparkRedshiftBuild.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2015 Databricks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.scalastyle.sbt.ScalastylePlugin.rawScalastyleSettings
+import sbt._
+import sbt.Keys._
+import sbtsparkpackage.SparkPackagePlugin.autoImport._
+import scoverage.ScoverageSbtPlugin
+
+object SparkRedshiftBuild extends Build {
+  val hadoopVersion = settingKey[String]("Hadoop version")
+
+  // Define a custom test configuration so that unit test helper classes can be re-used under
+  // the integration tests configuration; see http://stackoverflow.com/a/20635808.
+  lazy val IntegrationTest = config("it") extend Test
+
+  lazy val root = Project("spark-redshift", file("."))
+    .configs(IntegrationTest)
+    .settings(net.virtualvoid.sbt.graph.Plugin.graphSettings: _*)
+    .settings(Project.inConfig(IntegrationTest)(rawScalastyleSettings()): _*)
+    .settings(Defaults.itSettings: _*)
+    .settings(
+      name := "spark-redshift",
+      organization := "com.databricks",
+      version := "0.4.1-SNAPSHOT",
+      scalaVersion := "2.10.4",
```

Comment:
Doesn't have to be here, but in a followup can we make sure to cross publish 2.11?